test/mocha.opts

```diff
@@ -0,0 +1,3 @@
+--colors
+--reporter progress
```

Comment:
May be able to remove this

Reply:
Removing reporter line tests still do still pass